packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js

```diff
@@ -189,13 +189,8 @@ export default /* @ngInject */ ($stateProvider) => {
       },
       breadcrumb: () => null,
       trackingPrefix: () => 'vps::detail::dashboard',
-      canDisplayVpsAnnouncementBanner: /* @ngInject */ (ovhFeatureFlipping) => {
-        const vpsAnnouncementBannerId = 'vps:vps-announcement-banner';
```

Comment:
`vps:vps-announcement-banner` is not used anymore ?

Reply:
No, i will reuse it later.
It's a temporary changes time to found a way to deals things properly with API side.